### PR TITLE
Added client side and server side restrictions

### PR DESCRIPTION
**Pull request by PurpleDragon1999**

### CODE REVIEW

File: api/controllers/round.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class Round {
+    constructor(){
+
+    }
+
+    async createRound(req,res){
+        try{
+            console.log(req.body);
+            console.log(req.params);
+            let roundObj = {
```
Comment (sonalichawla16) [inline api/controllers/round.js:12]: why console logs ???

File: api/controllers/round.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            };
+            console.log(roundObj);
+            let createdRound = await roundModel.save(req.body);
+            return res.send({
+                success: true,
+                payload: {
+                    body: roundObj,
+                    message: "created interview successfully"
+                }
+            });
```
Comment (sonalichawla16) [inline api/controllers/round.js:24]: incorrected casing

File: web/src/app/login/login.component.ts [hunk 3]
```diff
@@ -50,9 +50,19 @@ export class LoginComponent implements OnInit {
...
   checkAccount() {
     this.loggedIn = !!this.authService.getAccount();
   }
+
+  getProfile() {
+    this.http.get(GRAPH_ENDPOINT).toPromise()
+      .then(profile => {
+          this.profile = profile;
+          console.log(profile);
+      });
```
Comment (sonalichawla16) [inline web/src/app/login/login.component.ts:63]: console logs ??

File: web/src/index.html [hunk 4]
```diff
@@ -11,6 +11,7 @@
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/3a2aec7bb0.js" crossorigin="anonymous"></script>
 </head>
```
Comment (sonalichawla16) [inline web/src/index.html:15]: Assure that we are using the same font throughtout

Comment (PurpleDragon1999) [reply web/src/index.html:15]: Yes Sonali, actually this is for using font-awesome icons.

**Comment by sonalichawla16** [comment]
Conflicts

**Pull request closed without merging**